google_compute_engine/accounts/accounts_daemon.py

```diff
@@ -29,6 +29,10 @@
 from google_compute_engine.accounts import accounts_utils
 
 LOCKFILE = constants.LOCALSTATEDIR + '/lock/google_accounts.lock'
+DEFAULT_USERADD_CMD = 'useradd -m -s /bin/bash -p * {user}'
+DEFAULT_USERDEL_CMD = 'userdel -r {user}'
+DEFAULT_USERMOD_CMD = 'groupadd {group}'
+DEFAULT_GROUPADD_CMD = 'groupadd {group}'
```

Comment:
I missed this in my last review comment - since you only need these variables in the utils class, you can remove their definitions in this file and just have them default to `None`.